nvflare/app_common/workflows/base_fedavg.py

```diff
@@ -27,7 +28,7 @@
 from .model_controller import ModelController
 
 
-class BaseFedAvg(ModelController):
+class BaseFedAvg(ModelController, WFControllerSpec):
```

Comment:
Since `ModelController` already inherit `WFControllerSpec` we don't need to duplicate it here?

Reply:
Agree this is redundant, however one of the goals was to expose the interface to users so they are not required to only look into the internal `ModelController` implementation. @chesterxgchen